clients/go/cmd/zbctl/main_test.go

```diff
@@ -48,86 +48,103 @@ type integrationTestSuite struct {
+		cmd:        strings.Fields("--insecure deploy testdata/model.bpmn testdata/job_model.bpmn --resourceNames=model.bpmn,job.bpmn"),
 		goldenFile: "testdata/deploy.golden",
 		jsonOutput: true,
 	},
 	{
 		name:       "create instance",
-		setupCmds:  []string{"--insecure deploy testdata/model.bpmn"},
-		cmd:        "--insecure create instance process",
+		setupCmds:  [][]string{strings.Fields("--insecure deploy testdata/model.bpmn")},
+		cmd:        strings.Fields("--insecure create instance process"),
 		goldenFile: "testdata/create_instance.golden",
 		jsonOutput: true,
 	},
 	{
-		name:       "create worker",
-		setupCmds:  []string{"--insecure deploy testdata/job_model.bpmn", "--insecure create instance jobProcess"},
-		cmd:        "create --insecure worker jobType --handler echo",
+		name: "create worker",
+		setupCmds: [][]string{
+			strings.Fields("--insecure deploy testdata/job_model.bpmn"),
+			strings.Fields("--insecure create instance jobProcess"),
+		},
+		cmd:        strings.Fields("create --insecure worker jobType --handler echo"),
 		goldenFile: "testdata/create_worker.golden",
 	},
 	{
 		name:       "empty activate job",
-		cmd:        "--insecure activate jobs jobType --maxJobsToActivate 0",
+		cmd:        strings.Fields("--insecure activate jobs jobType --maxJobsToActivate 0"),
 		goldenFile: "testdata/empty_activate_job.golden",
 		jsonOutput: true,
 	},
 	{
-		name:       "single activate job",
-		setupCmds:  []string{"--insecure deploy testdata/job_model.bpmn", "--insecure create instance jobProcess"},
-		cmd:        "--insecure activate jobs jobType --maxJobsToActivate 1",
+		name: "single activate job",
+		setupCmds: [][]string{
+			strings.Fields("--insecure deploy testdata/job_model.bpmn"),
+			strings.Fields("--insecure create instance jobProcess"),
+		},
+		cmd:        strings.Fields("--insecure activate jobs jobType --maxJobsToActivate 1"),
 		goldenFile: "testdata/single_activate_job.golden",
 		jsonOutput: true,
 	},
 	{
-		name:       "double activate job",
-		setupCmds:  []string{"--insecure deploy testdata/job_model.bpmn", "--insecure create instance jobProcess", "--insecure create instance jobProcess"},
-		cmd:        "--insecure activate jobs jobType --maxJobsToActivate 2",
+		name: "double activate job",
+		setupCmds: [][]string{
+			strings.Fields("--insecure deploy testdata/job_model.bpmn"),
+			strings.Fields("--insecure create instance jobProcess"),
+			strings.Fields("--insecure create instance jobProcess"),
+		},
+		cmd:        strings.Fields("--insecure activate jobs jobType --maxJobsToActivate 2"),
 		goldenFile: "testdata/double_activate_job.golden",
 		jsonOutput: true,
 	},
+	{
+		name:       "send message with a space",
+		setupCmds:  [][]string{strings.Fields("--insecure deploy testdata/start_event.bpmn")},
+		cmd:        []string{"--insecure", "publish", "message", "Start Process", "--correlationKey", "1234"},
+		goldenFile: "testdata/publish_message_with_space.golden",
+		jsonOutput: true,
```

Comment:
I'm thinking it might be simpler to add this as a separate test case instead of changing all of the other tests cases and suite code just to cope with this specific test case. If we ever decide to remove this "assurance" and delete the test, then we'll probably forget to revert the other changes. Especially, since this isn't testing Zeebe functionality, it's more a functionality of the Cobra library, so I'm not sure that changing the all the other cases for something that isn't implemented in Zeebe is very robust. Wdyt? Although I'm not sure if writing a separate test for this will add to much duplication. I don't think so but I haven't tried to write it

Reply:
Maybe I'm being overzealous, it'll probably be fine